docs/DeveloperGuide.md

```diff
@@ -171,6 +171,50 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### The onboarding guide
+
+This section describes the implementation of the onboarding guide subsystem.
```

Comment:
I think its better if you call it a system or component instead of a subsystem.

Reply:
Fixed